packages/account/src/Constants/routes-config.js

```diff
@@ -141,11 +141,9 @@ let routesConfig;
 // For default page route if page/path is not found, must be kept at the end of routes_config array
 const route_default = { component: Page404, getTitle: () => localize('Error 404') };
 
-const getRoutesConfig = ({ is_appstore }) => {
-    if (!routesConfig) {
-        routesConfig = initRoutesConfig({ is_appstore });
-        routesConfig.push(route_default);
-    }
+const getRoutesConfig = ({ is_appstore }, is_social_signup) => {
+    routesConfig = initRoutesConfig({ is_appstore }, is_social_signup);
+    routesConfig.push(route_default);
```

Comment:
since the is_social_signup takes time to has value, we need to update after it receive value.